packages/lib/logger.ts

```diff
@@ -3,7 +3,7 @@ import { Logger } from "tslog";
 import { IS_PRODUCTION } from "./constants";
 
 const logger = new Logger({
-  minLevel: !!process.env.NEXT_PUBLIC_DEBUG ? 2 : 4,
+  minLevel: !!process.env.NEXT_PUBLIC_DEBUG ? 0 : 0,
```

Comment:
why do we set this always to 0 now? 

If that's intended then we can just put 0 here